src/core/core.defaults.js

```diff
@@ -39,13 +57,49 @@ export class Defaults {
 		this.title = undefined;
 		this.tooltips = undefined;
 		this.doughnut = undefined;
+		this._routes = {};
 	}
 	/**
 	 * @param {string} scope
 	 * @param {*} values
 	 */
 	set(scope, values) {
-		return merge(this[scope] || (this[scope] = {}), values);
+		return merge(getScope(this, scope), values);
+	}
+
+	/**
+	 * Routes the named defaults to fallback to another scope/name
+	 * Examples:
+	 * 	defaults.route('elements.arc', 'backgroundColor', '', 'color')
+	 * 	defaults.route('elements.line', ['backgroundColor', 'borderColor'], '', 'color')
+	 * 	defaults.route('elements.customLine', ['borderWidth', 'tension'], 'elements.line', ['borderWidth', 'tension'])
+	 * @param {string} scope
+	 * @param {string[]} names
+	 * @param {string} targetScope
+	 * @param {string|string[]} targetNames
+	 */
+	route(scope, names, targetScope, targetNames) {
+		const scopeObject = getScope(this, scope);
+		const targetScopeObject = getScope(this, targetScope);
+		const targetNamesIsArray = isArray(targetNames);
+		names.forEach((name, index) => {
```

Comment:
this looks a bit complicated. it might be useful to add a comment here explaining what this is doing / how it works

Reply:
Tried to simplify it a bit and added some comments. How does it look now?